src/site/content/en/blog/how-we-build-webdev-and-use-web-components/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "web.dev engineering blog #1: How we build the site and use Web Components"
+date: 2020-04-09
+authors:
+  - samthor
+description: |
+  In this first post from the web.dev engineering team, learn about how we build the site—including
+  our use of Eleventy and Web Components.
+hero: hero.jpg
+alt: Wind turbines on a hill
+tags:
+- post # to show up in blog
+- engineering-blog
+---
+
+This is the first post in web.dev's engineering blog.
+Over the coming months, we hope to share actionable insights from our work—so watch for posts with the [Engineering Blog tag](/tags/engineering-blog/)!
+Here we'll be covering the build process for our static site and the (optional!) JavaScript behind our web components.
+
+web.dev provides content about building modern web experiences and allows you to [measure](/measure/) your site's performance.
+Savvy users may have realized that [our Measure page](/measure/) is just an interface for [Lighthouse](https://developers.google.com/web/tools/lighthouse), which is also available in Chrome's DevTools.
+Signing in to web.dev lets you run regular Lighthouse audits on your site so you can see how its score changes over time.
+I'll be revisiting the Measure page a bit later, as we think it's fairly special. 🎊
+
+## Introduction
+
+Fundamentally, web.dev is a static site that is generated from a collection of Markdown files.
+We've chosen to use [Eleventy](https://www.11ty.dev/) because it's a polished, extensible tool which makes it easy to turn Markdown into HTML.
+
+We also use modern JavaScript bundles that we only serve to browsers supporting [`type="module"`](https://medium.com/dev-channel/es6-modules-in-chrome-canary-m60-ba588dfb8ab7), which includes `async` and `await`.
+We also happily use features that are supported by evergreen browsers but not by a minority of older versions.
+Because we're a static site, JavaScript is _just not required_ to read our content.
+
+Once the build process—which involves generating static HTML and bundling our JavaScript with Rollup—is complete, web.dev can be hosted with a simple static server for testing.
+The site is _almost_ completely static, but we have a few special needs that still benefit from [a custom Node.js server](https://github.com/GoogleChrome/web.dev/blob/master/server.js).
```

Comment:
I think you can click on a line of code in github and press "Y" to generate a permalink.

I think we should do that instead of linking to the files in this way. I'm not sure if github will redirect if we move or rename or delete the files.